docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -40,9 +40,9 @@ a selection of packages.
 There are different distributions available:
 
 * {version} (recommended): +docker.elastic.co/package-registry/distribution:{version}+ - Selection of packages from the production repository released with the {version} version of the {stack}.
+* lite-{version}: +docker.elastic.co/package-registry/distribution:lite-{version}+ - Subset of the most commonly used packages from the production repository released with the {version} version of the {stack}. This image is a good candidate to start using Fleet on air-gapped environments.
```

Comment:
```suggestion
* lite-{version}: +docker.elastic.co/package-registry/distribution:lite-{version}+ - Subset of the most commonly used packages from the production repository released with {stack} {version}. This image is a good candidate to start using {fleet} in air-gapped environments.
```

Reply:
@dedemorton  Should I update also the item related to production versioned (line 42?) with this change ?

```
* {version} (recommended): +docker.elastic.co/package-registry/distribution:{version}+ - Selection of packages from the production repository released with {stack} {version}.
```

Reply:
Yes please!